src/config.rs

```diff
@@ -66,12 +66,18 @@ fn build_cli_parser<'a, 'b>() -> App<'a, 'b> {
                 .long("git-tag")
                 .help("Optional commit the updated version and create a git tag."),
         )
+        .arg(
```

Comment:
to not introduce a breaking change, let's change the flag to be: `generate-lockfile` and reverse the condition at the bottom